README.md

```diff
@@ -134,8 +135,37 @@ The renderer's `checksum` method will give you access to the checksum that has b
 For an example of how to attach this value to the DOM on the client side, see the example in the [template](#template) section below.
 
 
+### `setCacheStrategy`
+
+`setCacheStrategy({ get: ..., set: ... */ }) -> undefined`
+
+The default cache strategy provided by Rapscallion is a naive one.  It is synchronous and in-memory, with no cache invalidation or TTL for cache entries.
+
+However, `setCacheStrategy` is provided to allow you to integrate your own caching solutions.  The function expects an options argument with two keys:
+
+- `get` should accept a single argument, the key, and return a Promise resolving to a cached value.  If no cached value is found, the Promise should resolve to `null`.
```

Comment:
I should clarify the expected shape of the cache values in the docs, however!  Thanks for pointing it out.